File: hibernate-embedded-database-minimal/src/main/java/org/danwatt/hibernate/embedded/model/Person.java
package org.danwatt.hibernate.embedded.model;

import lombok.Data;

import javax.persistence.*;

@Data
@Entity
@Table(name = "person")
public class Person {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    @Column(name = "name")
    private String name;

}

File: hibernate-kotlin/src/main/kotlin/org.danwatt.hibernate.embedded.model/Person.kt
package org.danwatt.hibernate.embedded.model

import javax.persistence.*

@Entity
@Table(name = "person")
data class Person(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        var id: Int? = null,
        @Column(name = "name",nullable = false)
        var name: String
)